models/public/ssdlite_mobilenet_v2/ssdlite_mobilenet_v2.md

```diff
@@ -0,0 +1,72 @@
+# ssdlite_mobilenet_v2
+
+## Use Case and High-Level Description
+
+The `ssdlite_mobilenet_v2` is used for object detection. For details, see [paper](https://arxiv.org/pdf/1801.04381.pdf) MobileNetV2: Inverted Residuals and Linear Bottlenecks.
```

Comment:
The first sentence isn't quite grammatical. It should be one of:

* The `ssdlite_mobilenet_v2` model is...
* `ssdlite_mobilenet_v2` is...